zerver/openapi/zulip.yaml

```diff
@@ -15208,6 +15208,12 @@ components:
           description: |
             ## Rate limit exceeded
 
+            The `retry-after` parameter in the response indicates how many seconds
+            the client must wait before making additional requests.
+
+            **Changes**: New in Zulip 4.0 (feature level 36), the `code` field is
+            present in rate limit errors.
```

Comment:
I tweaked this changes field to be more readable before merging.

Reply:
Now it's "**Changes**: The `code` field was not present in rate
limit errors before Zulip 4.0 (feature level 36)."